src/codeGenerator.ts

```diff
@@ -217,6 +224,7 @@ export class CodeGenerator {
 
   private _generateActionCall(action: Action): string {
     switch (action.name)  {
+      case 'closeBrowser':
```

Comment:
It seems like you write directly the "browser.close()" to the output. So probably we don't need the Action?

Reply:
oh, i forgot to remove it, thanks!